gems/pending/util/postgres_admin.rb

```diff
@@ -12,6 +12,10 @@ def self.data_directory
     Pathname.new(ENV.fetch("APPLIANCE_PG_DATA"))
   end
 
+  def self.template_directory
+    Pathname.new(ENV.fetch("APPLIANCE_TEMPLATE_DIRECTORY"))
+  end
```

Comment:
I know, PostgresAdmin has been the single areas that fetch ENV variables.
It's why I'm naming the ENV variables with APPLIANCE_* so that when we figure out where these belong or a better way, it's easier to track them down.